react/index.html

```diff
@@ -11,6 +11,7 @@
     <script type="module" src="/src/main.jsx"></script>
     <script>
       (function () {
+        // @NOTE: In order to load Web Chat, you have to set the PLATFORM_ID and DOMAIN in the script.
```

Comment:
We can link dev docs here as well - https://developers.helpshift.com/web-chat/getting-started/#manual-embed